.travis.yml

```diff
@@ -26,36 +26,32 @@ matrix:
             - clang-11
             - musl-tools
       rust:
-        - nightly-2021-12-15
+        - stable
       env:
         - RUST_BACKTRACE=1
         - CFLAGS_x86_64_fortanix_unknown_sgx="-isystem/usr/include/x86_64-linux-gnu -mlvi-hardening -mllvm -x86-experimental-lvi-inline-asm-hardening"
         - CC_x86_64_fortanix_unknown_sgx=clang-11
       before_script:
         - rustup target add x86_64-fortanix-unknown-sgx x86_64-unknown-linux-musl
+        - rustup toolchain add nightly
+        - rustup target add x86_64-fortanix-unknown-sgx --toolchain nightly
       script:
         - cargo test --verbose --locked --all --exclude sgxs-loaders && [ "$(echo $(nm -D target/debug/sgx-detect|grep __vdso_sgx_enter_enclave))" = "w __vdso_sgx_enter_enclave" ]
-        - cargo test --verbose --locked -p sgx-isa --features sgxstd --target x86_64-fortanix-unknown-sgx --no-run
         - cargo test --verbose --locked -p sgxs-tools --features pe2sgxs --bin isgx-pe2sgx
         - cargo test --verbose --locked -p dcap-ql --features link
         - cargo test --verbose --locked -p dcap-ql --features verify
-        - cargo build --verbose --locked -p aesm-client --target=x86_64-fortanix-unknown-sgx
-        - cargo build --verbose --locked -p aesm-client --target=x86_64-fortanix-unknown-sgx --features sgx-isa/sgxstd
-        - cargo test --locked -p nitro-attestation-verify
+        # uses backtrace, which still requires nightly on SGX
+        - cargo +nightly build --verbose --locked -p aesm-client --target=x86_64-fortanix-unknown-sgx
+        # uses sgxstd feature
+        - cargo +nightly build --verbose --locked -p aesm-client --target=x86_64-fortanix-unknown-sgx --features sgx-isa/sgxstd
+        - cargo +nightly test --verbose --locked -p sgx-isa --features sgxstd --target x86_64-fortanix-unknown-sgx --no-run
         # Unfortunately running `faketime '2021-09-10 11:00:00 GMT' cargo test -p nitro-attestation-verify` causes a segmentation
         #  fault while compiling. We only execute `faketime` during the tests
-        - cargo test --locked -p nitro-attestation-verify --no-run && faketime '2021-09-08 11:00:00 GMT' $(find ./target/debug/deps -name "nitro_attestation_verify*" -executable)
-        - cargo test --locked -p nitro-attestation-verify --no-run && faketime '2021-09-10 11:00:00 GMT' $(find ./target/debug/deps -name "nitro_attestation_verify*" -executable)
```

Comment:
This one fails for me locally

Reply:
That's surprising. There have been 5 PRs since that check was added, but I get the same.

Reply:
I think it's related to the find command used.